src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -79,6 +81,7 @@ export class QuickBar extends LitElement {
     this._done = false;
     this._focusSet = false;
     this._filter = "";
+    this._search = "";
```

Comment:
Added this so the previous filter would clear before launching the dialog again.